java-checks-test-sources/default/src/main/java/checks/api/undocumentedAPI/UndocumentedApiJava23.java

```diff
@@ -0,0 +1,61 @@
+package com.example.api.mypackage;
+
+/// Undocumented Api for Java 23 (and an example of bad documentation).
+public class UndocumentedApiJava23 {
+
+  public class BadNested {} // Noncompliant {{Document this public class by adding an explicit description.}}
+
+  ///  Documentated.
```

Comment:
Should this say "Documented"?
```suggestion
  ///  Documented.
```